packages/grafana-ui/src/components/OptionsUI/string.tsx

```diff
@@ -24,7 +24,7 @@ export const StringValueEditor: React.FC<FieldConfigEditorProps<string, StringFi
         onChange(evt.currentTarget.value.trim() === '' ? undefined : evt.currentTarget.value);
       }
     },
-    [onChange]
+    [item.settings?.useTextarea, onChange]
```

Comment:
nit: Best practices is to destruct nested props and use them in the dependencies.

Reply:
same https://github.com/grafana/grafana/pull/28026#discussion_r504646492